src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1263,11 +1278,12 @@ private void InitializeHelix()
 
             headLight = new DirectionalLight3D
             {
-                Color = System.Windows.Media.Color.FromRgb(128, 128, 128),
-                Direction = new Vector3D(0, 0, 1),
+                Color = System.Windows.Media.Color.FromArgb(255,255, 0, 0),
```

Comment:
this is a mistake, I was trying to see if the headlight was rendering at all - **it is not** 😢